main.go

```diff
@@ -92,13 +93,20 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
+	gvkLabelMap := map[schema.GroupVersionKind]util.Selector{
```

Comment:
Can we limit to other resources? e.g.: configmap
What resources do we watch in managedcluster namespaces? I am thinking we may limit the deployment/statefulset/configmap/secret etc to the default namespace

Reply:
add other resources.
also filter manifestwork, only cache those which has our owner label

Reply:
Can you add `service` as well?

Reply:
As we discussed in slack DM, we need to make sure that the filter we added doesn't impact the watch behavior, it looks like we can only watch resources in the cache.

Reply:
@morvencao no, it can work other resources which not claimed.  I have verified that.
@clyang82 svc added